docs/features/techdocs/using-cloud-storage.md

```diff
@@ -165,112 +136,62 @@ techdocs:
       bucketName: 'name-of-techdocs-storage-bucket'
 ```
 
-**2.1.3 Create the `TechDocs` user**
-
-This user will be used to interact with your bucket, it will only have
-permissions to **get - put** objects.
-
-In the IAM console, do the following:
-
-- Create a new user, `TechDocs`
-- Note down the TechDocs User credentials
-- Note down the Amazon Resource Name (ARN) for the TechDocs user. In the IAM
-  console, select the TechDocs user, and you can find the user ARN in the
-  Summary tab.
-
-**2.2 Attach policies**
```

Comment:
Sorry about removing all these existing instructions of creating an IAM user with user and bucket policies. I think we should just refer to the official documentation, so that they are always latest and most secure.